app/src/main/res/values/strings.xml

```diff
@@ -1315,6 +1315,14 @@
     <string name="title_nsfollow_url">Nightscout Follow URL</string>
     <string name="summary_nsfollow_download_treatments">Also download treatments from Nightscout as follower</string>
     <string name="title_nsfollow_download_treatments">Download Treatments</string>
+    <string name="title_connectfollow_user">CareLink Username</string>
+    <string name="summary_connectfollow_user">CareLink login username</string>
+    <string name="title_connectfollow_pass">CareLink Password</string>
+    <string name="summary_connectfollow_pass">CareLink login password</string>
+    <string name="title_connectfollow_country">CareLink Country</string>
+    <string name="summary_connectfollow_country">Two letter country ISO code</string>
```

Comment:
Settings shows a list of countries, not ISO country codes. Hence, this string is not correct.

Reply:
Mixed up with wizard datasource configuration description, corrected.